### PR TITLE
chore(docs): indicate avatar as experimental

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

**Review by rachelbt — APPROVED** [review]
👍

File: apps/docs/_includes/navigation.njk [hunk 1]
```diff
@@ -2,13 +2,21 @@
 <vwc-nav>
   <vwc-nav-disclosure label="Design System" open>
     {%  for item in designs | sort(false, false, 'title') %}
-    <vwc-nav-item href="{{ ('/designs/' + item.title) | url }}" text="{{ item.title | replace("-", " ") | capitalize }}" {% if ('/designs/' + item.title + '/') == page.url %}aria-current="page"{% endif %}></vwc-nav-item>
+    <vwc-nav-item
+			href="{{ ('/designs/' + item.title) | url }}"
+			text="{{ item.title | replace("-", " ") | capitalize }}
+				{% if (item.status) == "alpha" %}🧪{% endif %}}"
```
Comment (yinonov) [inline apps/docs/_includes/navigation.njk:8]:
```suggestion
				{% if (item.status) == "alpha" %}🧪{% endif %}"
```

File: apps/docs/_includes/navigation.njk [hunk 2]
```diff
@@ -2,13 +2,21 @@
 <vwc-nav>
   <vwc-nav-disclosure label="Design System" open>
     {%  for item in designs | sort(false, false, 'title') %}
-    <vwc-nav-item href="{{ ('/designs/' + item.title) | url }}" text="{{ item.title | replace("-", " ") | capitalize }}" {% if ('/designs/' + item.title + '/') == page.url %}aria-current="page"{% endif %}></vwc-nav-item>
+    <vwc-nav-item
+			href="{{ ('/designs/' + item.title) | url }}"
+			text="{{ item.title | replace("-", " ") | capitalize }}
+				{% if (item.status) == "alpha" %}🧪{% endif %}"
```
Comment (yinonov) [inline apps/docs/_includes/navigation.njk:8]:
```suggestion
			text="{{ item.title | replace("-", " ") | capitalize }}{% if (item.status) == "alpha" %}🧪{% endif %}"
```

**Review by rachelbt — CHANGES_REQUESTED** [review]
the side navigation is broken :( 
image

**Review by rachelbt — DISMISSED** [review]
YES!! 
Look s great and I love the status addition in the components.json, much more systematic! 👍

File: apps/docs/_includes/navigation.njk [hunk 3]
```diff
@@ -2,13 +2,19 @@
 <vwc-nav>
   <vwc-nav-disclosure label="Design System" open>
     {%  for item in designs | sort(false, false, 'title') %}
-    <vwc-nav-item href="{{ ('/designs/' + item.title) | url }}" text="{{ item.title | replace("-", " ") | capitalize }}" {% if ('/designs/' + item.title + '/') == page.url %}aria-current="page"{% endif %}></vwc-nav-item>
+    <vwc-nav-item
+			href="{{ ('/designs/' + item.title) | url }}"
+			text="{{ item.title | replace("-", " ") | capitalize }} {% if (item.status) == "alpha" %}🧪{% endif %}"
```
Comment (olaf-k) [inline apps/docs/_includes/navigation.njk:7]: `title="This component is experimental"` would help people wondering what the emoji means.

File: apps/docs/_includes/navigation.njk [hunk 4]
```diff
@@ -2,13 +2,19 @@
...
+			{% if ('/designs/' + item.title + '/') == page.url %}aria-current="page"{% endif %}></vwc-nav-item>
     {% endfor %}
   </vwc-nav-disclosure>
 
   <vwc-nav-disclosure label="Components" open>
     {%  for item in components | sort(false, false, 'title') %}
-    <vwc-nav-item href="{{ ('/components/' + item.title) | url }}" text="{{ item.title | replace("-", " ") | capitalize }}" {% if ('/components/' + item.title + '/') == page.url %}aria-current="page"{% endif %}></vwc-nav-item>
+    <vwc-nav-item
+			href="{{ ('/components/' + item.title) | url }}"
+			text="{{ item.title | replace("-", " ") | capitalize }}	{% if (item.status) == "alpha" %}🧪{% endif %}"
```
Comment (olaf-k) [inline apps/docs/_includes/navigation.njk:16]: (same remark)

**Review by olaf-k — CHANGES_REQUESTED** [review]

File: apps/docs/_includes/navigation.njk (hunk 3 above)
Comment (yinonov) [reply apps/docs/_includes/navigation.njk:7]: added 
image

File: apps/docs/_includes/navigation.njk (hunk 4 above)
Comment (yinonov) [reply apps/docs/_includes/navigation.njk:16]: add
image

File: apps/docs/_includes/navigation.njk (hunk 3 above)
Comment (yinonov) [reply apps/docs/_includes/navigation.njk:7]: it's more relevant to components at this point

**Review by rinaok — CHANGES_REQUESTED** [review]
Screenshot 2022-11-03 at 11 17 41

Why is the banner not full width?

**Comment by yinonov** [comment]
> Screenshot 2022-11-03 at 11 17 41
> 
> Why is the banner not full width?

thanks, resolved

**Review by rinaok — CHANGES_REQUESTED** [review]
Something happened to the main page. It's not working.

**Comment by yinonov** [comment]
> Something happened to the main page. It's not working.

thanks, resolved

File: apps/docs/pages/component-pages.njk [hunk 5]
```diff
@@ -8,9 +7,24 @@ pagination:
...
+	{% endfor %}
+
+	{% if (component.status) == "alpha" %}
+<script type="module" src="{{ "/assets/modules/components/banner/index.js" | url }}"></script>
+	{% endif %}
+{% endblock %}
 
+{% if (component.status) == "alpha" %}
+	{% block banner %}
+<vwc-banner class="banner" text="Alpha: this is an initial phase of component testing. expect breaking changes." connotation="warning"></vwc-banner>
```
Comment (rinaok) [inline apps/docs/pages/component-pages.njk:24]:
```suggestion
<vwc-banner class="banner" text="Alpha: this is the initial phase of component testing. Expect breaking changes." connotation="warning"></vwc-banner>
```

**Review by rinaok — CHANGES_REQUESTED** [review]

**Review by olaf-k — COMMENTED** [review]
it looks like the banner covers the nav when resized horizontally. (happens on Chrome desktop but works well on iPhone.)
image

**Review by rinaok — APPROVED** [review]

**Comment by yinonov** [comment]
> it looks like the banner covers the nav when resized horizontally. (happens on Chrome desktop but works well on iPhone.) image

resolved

**Review by olaf-k — APPROVED** [review]

**Pull request merged**